seqr/models.py

```diff
@@ -1104,6 +1104,35 @@ class Meta:
         json_fields = ['gene_id', 'tpm']
 
 
+class RnaSeqSpliceOutlier(DeletableSampleMetadataModel):
+    STRAND_CHOICES = (
+        ('+', '5′ to 3′ direction'),
+        ('-', '3′ to 5′ direction'),
+        ('*', 'Any direction'),
+    )
+
+    p_value = models.FloatField()
+    z_score = models.FloatField()
+    chrom = models.CharField(max_length=2)
+    start = models.IntegerField()
+    end = models.IntegerField()
+    strand = models.CharField(max_length=1, choices=STRAND_CHOICES)  # "+", "-", or "*"
+    gene_name = models.CharField(max_length=20)
```

Comment:
the base class already has a gene_id field, that should be sufficient

Reply:
I removed the `gene_name` field. Do we need to verify that the `gene_id` exists in the seqr DB and that the `gene_name` matches the DB info? Or do we not use the gene name for display?